docs/root.js

```diff
@@ -26,17 +27,17 @@ const theme = {
 
 export const Root = props =>(
   <StyleProvider components={components} theme={theme}>
-  <div style={{
-    marginLeft: 'auto',
-    marginRight: 'auto',
-    paddingLeft: '32px',
-    paddingRight: '32px',
-    paddingTop: '32px',
-    paddingBottom: '32px',
-    maxWidth: '1000px'
-  }}>
-    {props.children}
-  </div>
+    <div style={{
+      marginLeft: 'auto',
+      marginRight: 'auto',
+      paddingLeft: '32px',
+      paddingRight: '32px',
+      paddingTop: '32px',
+      paddingBottom: '32px',
+      maxWidth: '1000px'
+    }}>
+      {props.children}
+    </div>
```

Comment:
Try not to commit stuff like this. There's no prettier config in this repo, but there is a format script to align with the project style.